config/zones/GB-NIR.yaml

```diff
@@ -229,11 +229,12 @@ fallbackZoneMixes:
         unknown: 0.0
         wind: 0.34781314616395964
 parsers:
-  consumption: IE.fetch_consumption
-  consumptionForecast: IE.fetch_consumption_forecast
+  consumption: SMARTGRIDDASHBOARD.fetch_consumption
+  consumptionForecast: SMARTGRIDDASHBOARD.fetch_consumption_forecast
   price: ENTSOE.fetch_price
-  production: IE.fetch_production
-  productionPerModeForecast: IE.fetch_wind_forecasts
+  production: SMARTGRIDDASHBOARD.fetch_production
+  productionPerModeForecast: SMARTGRIDDASHBOARD.fetch_wind_forecasts
+  generationForecast: SMARTGRIDDASHBOARD.fetch_generation
```

Comment:
Apparently we don't have a `generation` type for the parsers functions? Only `generationForecast`?